examples/nextjs-sceduler/.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_YORKIE_API_ADDR='http://localhost:8080'
+NEXT_PUBLIC_YORKIE_API_KEY=''
```

Comment:
You need to newline here to keep POSIX standard. 

> **3.206 Line**
A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

Reference:

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline